fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateUpdateRDFSourcePersister.java

```diff
@@ -17,40 +17,51 @@
  */
 package org.fcrepo.persistence.ocfl.impl;
 
+import static org.fcrepo.kernel.api.operations.ResourceOperationType.CREATE;
+import static org.fcrepo.kernel.api.operations.ResourceOperationType.UPDATE;
+import static org.fcrepo.persistence.ocfl.OCFLPeristentStorageUtils.INTERNAL_FEDORA_DIRECTORY;
+import static org.fcrepo.persistence.ocfl.OCFLPeristentStorageUtils.relativizeSubpath;
+import static org.fcrepo.persistence.ocfl.OCFLPeristentStorageUtils.writeRDF;
+
 import org.fcrepo.kernel.api.operations.RdfSourceOperation;
+import org.fcrepo.kernel.api.operations.ResourceOperationType;
 import org.fcrepo.persistence.api.exceptions.PersistentStorageException;
 import org.fcrepo.persistence.ocfl.api.OCFLObjectSession;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
-
-import static org.fcrepo.kernel.api.operations.ResourceOperationType.CREATE;
-import static org.fcrepo.persistence.ocfl.OCFLPeristentStorageUtils.INTERNAL_FEDORA_DIRECTORY;
-import static org.fcrepo.persistence.ocfl.OCFLPeristentStorageUtils.relativizeSubpath;
-import static org.fcrepo.persistence.ocfl.OCFLPeristentStorageUtils.writeRDF;
+import java.util.HashSet;
+import java.util.Set;
 
 /**
  * This class implements the persistence of a new RDFSource
  *
  * @author dbernstein
  * @since 6.0.0
  */
-public class CreateRDFSourcePersister extends AbstractPersister<RdfSourceOperation> {
+public class CreateUpdateRDFSourcePersister extends AbstractPersister<RdfSourceOperation> {
```

Comment:
Would `RDFSourcePersister` be too minimal? I'm not sure if there are going to be other RDF source persisters... Seems like deleting would potentially cover all object types (although there would have to be special logic according to what the interaction model was).